python-frontend/src/main/java/org/sonar/python/semantic/ClassSymbolImpl.java

```diff
@@ -150,9 +151,14 @@ public boolean hasUnresolvedTypeHierarchy(boolean includeAmbiguousSymbols) {
     return Optional.empty();
   }
 
+  @Override
+  public boolean hasMetaClass() {
+    return hasMetaClass;
```

Comment:
It looks like in Python2 you define metaclasses with an attribute: https://docs.python.org/2.7/reference/datamodel.html#__metaclass__
Maybe we can have
```java
return hasMetaClass || membersByName.get("__metaclass__") != null
```